cogs/minecraft.py

```diff
@@ -59,5 +59,37 @@ async def serverinfo(self, ctx):
         
         await ctx.send(embed=embed)
         
+@client.command(aliases=['players', 'onlineplayers'])
+async def serverplayers(ctx):
+    status = None
+    status1 = None
+    try:
+        server = MinecraftServer.lookup("billysbasement.aternos.me")
+        status = server1.status()
+    except:
+        pass
+    try:
+        server1 = MinecraftServer.lookup("147.135.71.70:25592")
+        status1 = server1.status()
+    except:
+        pass
+    msg = ""
+    msg1 = ""
+    if status1 is not None:
+        for x in status1.players.sample:
+            msg1 += f"{x.name} "
```

Comment:
this will result as all player names in one line. adding a newline escape character (`\n`) can help.